src/pages/Service.js

```diff
@@ -53,13 +55,26 @@ export default function Service() {
     }, []);
 
     async function fetchService(path) {
-        fetch(path).then(res => res.json()).then(data => {
+        fetch(path).then(res => {
+            if (!res.ok) {
+                throw new Error(`Service ${service.name} not found`);
+            }
+            res.json()
```

Comment:
You need `return res.json()` here, because you are now defining a function body. The shorthand only works when you have one statement.